easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-5.0.1.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'PythonBundle'
+
+name = 'vsc-mympirun'
+version = '5.0.1'
+
+homepage = 'https://github.com/hpcugent/vsc-mympirun'
+description = """mympirun is a tool to make it easier for users of HPC clusters to
+run MPI programs with good performance."""
+
+# we build this to work with every python version
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+use_pip = False
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('setuptools', '46.1.1', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'checksums': ['7416e775d09a98aa0422fbe367d15ec0904e4f5aa01043e35bbcb98fa53038de'],
+    }),
+    ('future', '0.18.2', {
+        'checksums': ['b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d'],
+    }),
+    ('pbr', '5.4.4', {
+        'checksums': ['139d2625547dbfa5fb0b81daebb39601c478c21956dc57e2e07b74450a8c506b'],
+    }),
+    ('funcsigs', '1.0.2', {
+        'checksums': ['a7bb0f2cf3a3fd1ab2732cb49eba4252c2af4240442415b4abce3b87022a8f50'],
+    }),
+    ('mock', '3.0.5', {
+        'checksums': ['83657d894c90d5681d62155c82bda9c1187827525880eda8ff5df4ec813437c3'],
+    }),
+    ('IPy', '1.00', {
+        'modulename': 'IPy',
+        'checksums': ['2f2bf658a858d43868d8a4352b3889cf78c66e2ce678b300dcf518c9149ba621'],
+    }),
+    ('vsc-install', '0.15.0', {
+        'modulename': 'vsc.install',
+        'checksums': ['d6709feaf5451ec005381e5787a0086a35323efa4f6f0ead2b408cdffac15144'],
+    }),
+    ('vsc-base', '2.9.6', {
+        'modulename': 'vsc.utils',
+        'checksums': ['a62158022c36b5b456771ee8145d9ea3d299b301240e9724d04052e6aa0400b3'],
+    }),
+    (name, version, {
```

Comment:
@boegel now it's you who removed the comment from the previous version :)
or does it not apply anymore?